### ISSUE TITLE
Control+v does not work in Word 365 with NVDA 2020.4

### ISSUE THREAD

**Issue by andrew-l-d** [nvaccess/nvda#12110]
I am not at all convinced that NVDA is causing the problem, but since updating to NVDA 2020.4 on two machines I cannot use control+v to paste in MS Word.  The problem occurred on a third machine I am aware of.  If someone who has NVDA 2020.4 and Office 365 installed could eliminate NVDA as the cause, that would be most helpful.  The problem does not occur in Outlook or Excel.
### Steps to reproduce:
1.  Install NVDA 2020.4
2. Create or open a file in MS Word under Office 365.
3. Write some text.
4. Select some of the text.
5. Press control+c to copy.
6. Move to - say - a blank line and press control+v.
### Actual behavior:
No text appears.
### Expected behavior:
Copied text should be pasted.
#### NVDA installed/portable/running from source:
NVDA installed
#### NVDA version:
2020.4
#### Windows version:
Windows 10 Home
#### Name and version of other software in use when reproducing the issue:
MS Word
#### Other information about your system:
8gb of RAM.  
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
I did not notice the problem with NVDA 2020.3 or with alpha builds of 2020.4.  Tried a portable copy of NVDA 2020.2 just now and the problem persisted.  My concern is that NVDA 2020.4 may be corrupting something on installation, which seems unlikely.
#### If addons are disabled, is your problem still occuring?
Yes and the problem occurs when starting the computer without NVDA launching.
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No.

**Comment by leonardder** [comment]
1. Are you saying that the problem occurs with NVDA turned of?
2. In what language are you using NVDA?

**Comment by andrew-l-d** [comment]
1.  Yes, which tends to point to NVDA not being the cause of the problem.
2. English.

**Comment by Brian1Gaff** [comment]
Is anything else running that might hijack the keys?

Seems do to me since its a standard windows shortcut. Word d does have its 
own clipboard I think, so it would be interesting to know if anything is c
actuallyg
 y
going into it?
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by andrew-l-d** [comment]
No Brian, nothing seems to be highjacking control+v.  As mentioned, no problems in Outlook or Excel.  Control+v also works in Notepad etc.  I can paste in Word by using the Ribbon, so the Office clipboard is behaving as it should.
Feedback confirming or not the issue when running NVDA 2020.4 would provide some clarity.

**Comment by XLTechie** [comment]
This may be a stupid question, but can you be absolutely sure that nothing is 
being pasted?

If you type the following four lines:

This.
is.
a.
test.

Then you select the line that contains the word "is" (move to the line and 
press shift+down arrow), and press control+c.
Next, press control+v three times.
Last, move to the top of document (control+home), and press NVDA+down arrow 
(desktop layout), to read from cursor to end.

I get the following:

This.
is.
is.
is.
a.
test.

I'd like to here what you actually get, assuming it isn't that.
I have given exact keystrokes here, not to say you don't know what you're doing, 
but so you know exactly what I did.
I used the period after each word to provide a speech pause.

I'm using Word 2019, not 365, though there shouldn't be a difference.

**Comment by andrew-l-d** [comment]
It is not a stupid question, just doubting my competence (which I do several times a day).  I did your exercise for completeness.  Amusingly, I copied your lines and went into Word and automatically pressed control+v and nothing happened - had to go via the Ribbon.
A friend has reported no problems with V2020.4 and Word 365 and the third machine I mentioned originally has reverted to normal operation.  It is looking even less likely that NVDA is the culprit. I neglected to mention that control+v works in safe mode.  Microsoft spent an hour or so trying to track down the issue yesterday and I hope to hear back tomorrow.

**Comment by XLTechie** [comment]
These seem unlikely to help, but:

1. Try the Com Reg Fix tool (NVDA+n, t, r).
If pasting still doesn't work:

2. Try resetting NVDA's gestures (NVDA+n, p, n, alt+d, alt+y).
If that doesn't help:

3. For completeness: Press NVDA+control+r three times quickly, to reset to the 
NVDA factory default.

**Comment by XLTechie** [comment]
In part, it seems unlikely that those things will help, because this is probably 
not NVDA's problem. But your messages cast just enough doubt on that to make it 
uncertain.

It's a very weird issue.

**Comment by andrew-l-d** [comment]
The cause of the problem has been found.  MathType is the culprit.  They must have changed something recently, as I have had it installed for a couple of years.  Had to uninstall MathType.  Both MIcrosoft and I will follow up with Wiris.  MathPlayer is ok.
